src/cabal

```diff
@@ -1,3 +1,4 @@
+# EXTRAS_VERSION
```

Comment:
yes, <(...) is bash only, and moving the #! down means that it defaults to bash (at least on OSX), hence my hack accidentally worked. Will do it differently. Lucky you asked and I checked.